config/initializers/monkey_patches.rb

```diff
@@ -1,7 +0,0 @@
-# frozen_string_literal: true
-
-# Copyright 2015-2017, the Linux Foundation, IDA, and the
-# CII Best Practices badge contributors
-# SPDX-License-Identifier: MIT
-
-Dir[Rails.root.join('lib', 'ext', '**', '*.rb')].each { |file| require file }
```

Comment:
Why is this being removed?

In general I like removing unused code, but I view this a pre-set-up "escape valve" for the desperate.